gdk-pixbuf/src/auto/versions.txt

```diff
@@ -1,2 +1,2 @@
-Generated by gir (https://github.com/gtk-rs/gir @ 7047e1b433e2)
+Generated by gir (https://github.com/gtk-rs/gir @ ab56426172f9)
```

Comment:
I'll get rid of this noise (and the corresponding CI failure) before merging, don't worry about that